data/jsons/board_detect.json

```diff
@@ -36,6 +36,28 @@
        "match": [ "SALT BAY" ]
       }
      ]
+    },
+    {
+     "name": "intel-minnow-max-new-gpio-numbering",
```

Comment:
awful name, try to add a version or something we can refer. What happens if another new schema appears? -new-new? :-D

Add something like `intel-minnow-max-linux_gt_3.19`.